### DISCUSSION TITLE
Is there a blender version for moonray?

### DISCUSSION THREAD

**Discussion by RiceWitch** [dreamworksanimation/openmoonray#13] [Q&A]
Can't find Jack on it, I thought someone already made it but nope-

**Comment by thomasb9511** [comment]
https://github.com/dreamworksanimation/openmoonray/discussions/1#discussioncomment-5324619

**Reply by randypacker** [reply]
I'm not aware of a Blender plugin for MoonRay (yet).  We're with you on we believe many people are interested in this, and and perhaps pursuing it (plus there's a lot of great stuff happening in the Blender community).  We're following along with the Blender/Hydra integrations, as we do have a Hydra delegate that works in compatible DCC's supporting Hydra.

**Comment by Pasqualle** [comment]
https://devtalk.blender.org/t/moonray-and-cycles/28436

**Comment by fantomid** [comment]
Work in progress:
https://devtalk.blender.org/t/moonray-plugin-for-blender/35272/46
